datahub-web-react/src/app/useGetAuthenticatedUser.tsx

```diff
@@ -10,7 +10,7 @@ export function useGetAuthenticatedUser(skip?: boolean) {
     if (!userUrn) {
         throw new Error('Could not find logged in user.');
     }
-    const { data, error } = useGetMeQuery({ skip, fetchPolicy: 'cache-first' });
+    const { data, error } = useGetMeQuery({ skip, fetchPolicy: 'cache-and-network' });
```

Comment:
Curious - why is this change necessary now?

Reply:
cache-first wouldn't call network, so we wouldnt get updated settings